setup.py

```diff
@@ -21,7 +21,7 @@
     ],
     packages=find_packages(exclude=["tests"]),
     install_requires=[
-        "django>=1.8,<1.9",
+        "django>=1.8,<2.0",
```

Comment:
I assumed there would be a Django 1.12, but I just checked the release schedule and the next version will be 2.0. So 1.12 and 2.0 should have the exact same effect here.